src/appengine/handlers/jobs.py

```diff
@@ -97,11 +107,18 @@ def get(self):
             'result': result,
             'templates': templates,
             'fieldValues': {
-                'csrf_token': form.generate_csrf_token(),
-                'queues': queues,
-                'update_job_url': '/update-job',
-                'update_job_template_url': '/update-job-template',
-                'upload_info': gcs.prepare_blob_upload()._asdict(),
+                'csrf_token':
+                    form.generate_csrf_token(),
+                'fuzzers':
+                    data_handler.get_all_fuzzer_names_including_children(),
```

Comment:
This should be something like:
```python
fuzzers = [
        fuzzer.name for fuzzer in data_types.Fuzzer.query(projection=['name'])
]
```
get_all_fuzzer_names_including_children isn't the right one to use, as it includes child fuzzers. child fuzzer -> job associations are set to the parents, and not modifiable .

Reply:
done 👍